### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by martincostello**
Fix a number of minor typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ When there are machines which have access to both the public internet and the GH
 - `repo-name-list` _(optional)_
    A comma-separated list of repositories to be synced. Each entry follows the format of `repo-name`.
 - `repo-name-list-file` _(optional)_
-   A path to a file containing a newline separate listof repositories to be synced. Each entry follows te format of `repo-name`.
+   A path to a file containing a newline separated list of repositories to be synced. Each entry follows te format of `repo-name`.
```
Comment (ajaykn) [inline README.md:38]:
```suggestion
   A path to a file containing a newline separated list of repositories to be synced. Each entry follows the format of `repo-name`.
```

**Review by ajaykn — COMMENTED** [review]
Thanks for your contribution. 
Minor typo.

**Review by h2floh — APPROVED** [review]
Thanks for your contribution @martincostello!

**Review by ajaykn — APPROVED** [review]
❤️

**Pull request merged**